### PR TITLE
Create airflow file for drugs app

### PR DESCRIPTION
**Pull request by MeganStodel**

### CODE REVIEW

File: drugs-app-airflow.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from datetime import datetime, timedelta
+
+import airflow
+from airflow import DAG
+from airflow.contrib.operators.kubernetes_pod_operator import KubernetesPodOperator
+from airflow.utils.dates import days_ago
+
+# Define your docker image and the AWS role that will run the image (based on your airflow-repo)
+IMAGE = "593291632749.dkr.ecr.eu-west-1.amazonaws.com/airflow-bentham-app:v0.2.0"
```
Comment (MandarinDuck) [inline drugs-app-airflow.py]: This is using the Bentham work, is that correct?

File: drugs-app-airflow.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ROLE = "airflow_drugs_app"
+
+# Task arguments
+task_args = {
+    "depends_on_past": False,
+    "email_on_failure": True,
+    "retries": 3,
```
Comment (MandarinDuck) [inline drugs-app-airflow.py]: Best not to use retries and exp backoff if this is a first iteration.

File: drugs-app-airflow.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "owner": "meganstodel",
+    "email": ["megan.stodel@justice.gov.uk"],
+}
+
+# # # Define your DAG
+dag = DAG(
+    "drugs_app",
+    default_args=task_args,
+    description="Check s3 for new drug finds data, then add to database if present.",
+    start_date=datetime(2019, 2, 19, 2),
```
Comment (MandarinDuck) [inline drugs-app-airflow.py]: Not sure the 4th argument here will do anything, might be best to remove it.

File: drugs-app-airflow.py (hunk 1 above)
Comment (MeganStodel) [reply drugs-app-airflow.py]: No! It should be 
```suggestion
IMAGE = "593291632749.dkr.ecr.eu-west-1.amazonaws.com/airflow-drugs-app:v0.0.8"
```

File: drugs-app-airflow.py (hunk 2 above)
Comment (MeganStodel) [reply drugs-app-airflow.py]: OK, so...

```suggestion
```

File: drugs-app-airflow.py (hunk 3 above)
Comment (MeganStodel) [reply drugs-app-airflow.py]: As in the `2` or the whole `start_date` bit?

```suggestion
```

**Pull request merged**